.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Remove from the repo (`git rm`) and add to `.gitignore`

Reply:
Didn't understand, how come it isn't ignored by develop's gitignore and yet the folder doesn't exist there?